pkgs/applications/science/programming/scyther/cli.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, cmake, glibc, makeWrapper, fetchFromGitHub, flex, bison, source }:
+stdenv.mkDerivation { 
+  name = "scyther-cli-${source.rev}";
+  src = source;
+  preConfigure = ''
+    cd src
+    # Since we're not in a git dir, the normal command this project uses to create this file wouldn't work
```

Comment:
This works until the moment it doesn't. Please add some kind of assertion checking that upstream didn't change their method of producing this string in a couple of versions.